lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -55,6 +56,9 @@ interface State {
 
 // Styles.
 const AvailableDays = styled(FieldSet)`
+  display: flex;
+  gap: 4px;
+
```

Comment:
These labels and inputs aren't appearing centered to me, especially when there's a unavailable day 
![image](https://github.com/user-attachments/assets/815244a1-d536-49f2-b0ea-aef1f7b16090)

Reply:
I do see that on Firefox but not Chrome... It is actually the text that is slightly offset vertically, the icon and the checkbox alignment are fine.

Reply:
You see a similar offset with the regular input controls too:
<img width="117" alt="image" src="https://github.com/user-attachments/assets/c99f258e-1814-4d12-80dc-dbb7b19edda0">